core/jobs_test.py

```diff
@@ -789,9 +789,26 @@ def test_each_one_off_class_is_subclass_of_base_job_manager(self):
         for klass in jobs_registry.ONE_OFF_JOB_MANAGERS:
             self.assertTrue(issubclass(klass, jobs.BaseJobManager))
 
+    def test_is_abstract_method(self):
+        class TestMockAbstractClass(jobs.BaseJobManager):
+            """A sample Abstract Class."""
+            pass
+
+        mock_abstract_base_class = [
+            TestMockAbstractClass
+        ]
+        abstract_list_swap = self.swap(
+            jobs, 'ABSTRACT_BASE_CLASSES', mock_abstract_base_class)
+        assert_raises_regexp_abstract_function = self.assertRaisesRegexp(
+            Exception,
+            'Tried to directly create a job using the abstract base*')
+        with assert_raises_regexp_abstract_function, abstract_list_swap:
+            for klass in mock_abstract_base_class:
```

Comment:
This loop isn't require, we only have a single class. Use it directly.

Reply:
So instead of loop, should I write `mock_abstract_base_class[0]`? Because a list is mandatory, otherwise I get an error "argument of type 'type' is not iterable" if I don't use a loop and a list.

Reply:
You can directly use `TestMockAbstractClass`, right?

Reply:
Done.